### DISCUSSION TITLE
[Feature] Multi part upload for large files

### DISCUSSION THREAD

**Discussion by TheLonelinessOfHS** [immich-app/immich#6292] [Feature Request]
### The feature

Uploading big files will result in HTTP Error 413 if the Immich server is behind Cloudflare which has an upload size limit (not about nginx conf here). Please consider implementing multi-part upload for big files to address this issue. Thanks! 

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by schuhbacca** [comment]
Already an open feature request: https://github.com/immich-app/immich/discussions/1674
